src/config.ts

```diff
@@ -197,23 +207,27 @@ export namespace Config {
       }
     }
 
-    function readSettings(): Config | undefined {
+    function readSettings(): TConfig | undefined {
       output.appendLine(alloglot.ui.readingWorkspaceSettings)
       const workspaceSettings = vscode.workspace.getConfiguration(alloglot.config.root)
       const activateCommand = workspaceSettings.get<string>(alloglot.config.activateCommand)
+      const deactivateCommand = workspaceSettings.get<string>(alloglot.config.deactivateCommand)
       const languages = workspaceSettings.get<Array<LanguageConfig>>(alloglot.config.languages)
-      const settingsExist = !!(activateCommand || languages)
+      const verboseOutput = workspaceSettings.get<boolean>(alloglot.config.verboseOutput)
+      const settingsExist = !!(activateCommand || languages || deactivateCommand || verboseOutput)
       output.appendLine(alloglot.ui.workspaceConfigExists(settingsExist))
-      if (settingsExist) return { activateCommand, languages }
+      if (settingsExist) return { activateCommand, deactivateCommand, languages, verboseOutput }
       return undefined
     }
 
     return sanitizeConfig(readSettings() || readFallback() || empty)
   }
 
-  function sanitizeConfig(config: Config): Config {
+  function sanitizeConfig(config: TConfig): TConfig {
     return {
       activateCommand: config.activateCommand?.trim(),
+      deactivateCommand: config.deactivateCommand?.trim(),
+      verboseOutput: !!config.verboseOutput,
```

Comment:
Why doesn't this need a trim?

Reply:
`trim()` so that in case it's all spaces I won't try to run it.